src/components/LocaleSwitcherComponent.vue

```diff
@@ -30,7 +30,7 @@ export default {
 	data() {
 		return {
 			open: false,
-			locales: ['en-US', 'sv-SE'],
+			locales: ['en-US', 'sv-SE', 'es-ES'],
```

Comment:
Shouldn't this be `es-CO`? 🤔 
```suggestion
			locales: ['en-US', 'sv-SE', 'es-CO'],
```